diff --git a/dmAsync/connection.py b/dmAsync/connection.py
index 884c49f..863304b 100644
--- a/dmAsync/connection.py
+++ b/dmAsync/connection.py
@@ -684,7 +684,7 @@ class Connection:
             scrollable=scrollable,
             withhold=withhold,
         )
-        cursor = Cursor(self, impl, timeout, isolation_level)
+        cursor = Cursor(self, impl, timeout, False, isolation_level)
         return cursor
 
     async def _cursor_impl(
